presto-product-tests/src/main/java/com/facebook/presto/tests/TestGroups.java

```diff
@@ -42,6 +42,11 @@
     public static final String STRING_FUNCTIONS = "string_functions";
     public static final String MATH_FUNCTIONS = "math_functions";
     public static final String STORAGE_FORMATS = "storage_formats";
+    public static final String PROFILE_SPECIFIC_TESTS = "profilespecifictests";
```

Comment:
Please follow the `groups` naming convention. This particular group should be called `profile_specific_tests`.
Please reformat all the other test group names to follow this convention.